.circleci/config.yml

```diff
@@ -51,6 +51,29 @@ references:
       name: Check seed peers
       command: make check-seed-peers
 
+  fail_notification: &fail_notification
```

Comment:
I see similar to https://github.com/aeternity/epoch/blob/6d17782d7cf75fa373fef9866ad560a776dd6d7e/.circleci/config.yml#L190